grails-app/services/org/pih/warehouse/core/UserService.groovy

```diff
@@ -236,6 +236,13 @@ class UserService {
         return isSuperuser(currentUser) || (currentUser.getHighestRole(location) >= otherUser.getHighestRole(location))
     }
 
+    boolean userHasRoles(String userId, Collection roleTypes, String locationId) {
```

Comment:
Use `hasRoles` as the name here too. Although, I'd also be fine with 
* doesUserHaveRoles 
* isUserInRoles
* isUserInAllRoles
* isUserInAnyRole